assets/js/modules/analytics/datastore/properties.js

```diff
@@ -476,8 +476,15 @@ const baseSelectors = {
 	getPropertiesIncludingGA4: createRegistrySelector( ( select ) => ( state, accountID ) => {
 		let properties = select( STORE_NAME ).getProperties( accountID );
 
+		if ( properties === undefined ) {
+			return undefined;
+		}
+
 		if ( select( MODULES_ANALYTICS_4 ) ) {
```

Comment:
I know this is not relevant for this ticket, but let's check that the `ga4setup` feature is enabled instead of checking its datastore:

```suggestion
		if ( isFeatureEnabled( 'ga4setup' ) ) {
```

Reply:
Makes sense @eugene-manuilov !  IIRC you said to do the same change elsewhere on another PR :smile: